code/game/objects/items/devices/lightreplacer.dm

```diff
@@ -52,6 +52,7 @@
 	flags_1 = CONDUCT_1
 	slot_flags = SLOT_BELT
 	origin_tech = "magnets=3;engineering=4"
+	force = 8

```

Comment:
Its such a clumsy weapon, it should do half damage as this if anything.

Reply:
clumsy is of no consequence when applied directly to head